### ISSUE TITLE
Signups and Reportbacks support

### ISSUE THREAD

**Issue by aaronschachter** [DoSomething/northstar-js#13]
Blocked by https://github.com/DoSomething/northstar/issues/429

To be used by [Gambit CampaignBot](http://github.com/dosomething/gambit/issues/606):
- [x] `getSignups(campaignId, userId)` -- need to query by Campaign and User ID. 
- [x] Add something like a Signup `current` property to determine if it's current (inspect the [`campaign_run.current` property](https://github.com/DoSomething/phoenix/blob/dev/documentation/endpoints/signups.md#retrieve-all-signups))
- [ ] `createSignup(campaignId, userId)`
- [ ] `upsertReportback(campaignId, userId, reportbackData)`

**Comment by aaronschachter** [comment]
No longer relevant - Gambit posts Signups and Reportbacks to the Phoenix API (not Northstar).
